appserver/orb/orb-iiop/src/main/java/org/glassfish/enterprise/iiop/impl/IIOPSSLSocketFactory.java

```diff
@@ -105,10 +106,9 @@
     private static final String PERSISTENT_SSL = "PERSISTENT_SSL";
 
     private static final int BACKLOG = 50;
-    
+
     private static final String SO_KEEPALIVE = "fish.payara.SOKeepAlive";
```

Comment:
I think the case of this property is so complicated, it will lead to issues.
May I suggest ```fish.payara.so-keepalive```